boxsdk/object/api_json_object.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+from collections import Mapping
+from abc import ABCMeta
+import six
+
+from .base_api_json_object import BaseAPIJSONObject, BaseAPIJSONObjectMeta
+
+
+class APIJSONObjectMeta(BaseAPIJSONObjectMeta, ABCMeta):
```

Comment:
This is an empty ABCMeta which is a little strange to me.  I don't follow the comment about avoiding conflicts.  We get BetaAPIJSONObjectMeta thru BaseAPIJSONObject below in APIJSONObject... so, I'm not sure what this is needed for.